src/Stratis.Bitcoin.Tests.Common/KnownNetworks.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NBitcoin;
+using NBitcoin.Networks;
+using Stratis.Bitcoin.Networks;
+
+namespace Stratis.Bitcoin.Tests.Common
+{
+    public static class KnownNetworks
```

Comment:
I am wondering now if KnownNetworks should not be in the Networks project

Reply:
Remember that `KnownNetworks` is just a wrapper for the `NetworkRegistration` class. `NetworkRegistration` manages the registered networks in its own internal collection called `registeredNetworks`. We can even look at deleting this over time.

Reply:
yeah also this will force us to develop in a network agnostic way so ok lets keep it out.